### DISCUSSION TITLE
enhance the file transfer

### DISCUSSION THREAD

**Discussion by shabakett** [rustdesk/rustdesk#1941] [General]
enhance the file transfer by:

 - showing directory browser dialog for directory selection
 - allow drag and drop files between local and remote machines

**Comment by test2a** [comment]
i would add, clicking on file transfer seems to open a new connection and it is not instantaneous like anydesk

**Comment by Vertganti** [comment]
Has support for drag&drop file transfer been added? I haven't found a way to make it work and there is high demand for it within our network.
